src/panels/lovelace/common/process-config-entities.ts

```diff
@@ -34,9 +34,11 @@ export const processConfigEntities = <T extends EntityConfig>(
         throw new Error(`Invalid entity specified at position ${index}.`);
       }
 
-      if (!isValidEntityId(config.entity)) {
+      if (!isValidEntityId((config as EntityConfig).entity!)) {
         throw new Error(
-          `Invalid entity ID at position ${index}: ${config.entity}`
+          `Invalid entity ID at position ${index}: ${
+            (config as EntityConfig).entity
+          }`
```

Comment:
I am not exactly sure what the best way to do this is. With the cast, it compains that entity doesn't exist in Type T.

Even if I add `"entity" in config` to the if statement.